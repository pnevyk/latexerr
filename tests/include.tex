\documentclass{article}

\begin{document}

\barone
\include{included}
\bartwo
\include{included}
\barthree

\end{document}
