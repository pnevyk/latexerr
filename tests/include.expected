File: include.tex

Error on line 5: Unknown command \barone.
Error on line 7: Unknown command \bartwo.
Error on line 9: Unknown command \barthree.

File: included.tex

Error on line 1: Unknown command \foo.
